Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -532,10 +544,10 @@ - (BOOL)hasBottomFooter
     return !self.viewItem.shouldHideFooter && !shouldFooterOverlayMedia;
 }
 
-- (void)insertAnyTextViewsIntoStackView:(NSArray<UIView *> *)textViews
+- (BOOL)insertAnyTextViewsIntoStackView:(NSArray<UIView *> *)textViews
```

Comment:
TIOLI: I think the code that calls this method could be clearer if this method returns a `UIStackView?` instead of a BOOL and the caller optionally does the insertion.

At that point, something like `buildTextStackView:` would probably be a better name.

Reply:
It's a tough choice.  `configureViews` is already _huge_.  If I get a chance, I'd like to decompose it a bit, but it's always going to be awkwardly large.  I'm wary of making it even longer.